tests/metrics/test_fidelity.py

```diff
@@ -50,20 +51,24 @@ def test_perturbation_metrics():
     model = generate_timeseries_model(input_shape, nb_labels)
     explanations = np.random.uniform(0, 1, x.shape)
 
-    for step in [-1, 2, 10]:
-        for max_percentage_perturbed in [0.2, 1.0]:
-            for baseline_mode in [0.0, "zero", "inverse", "negative"]:
-                score_insertion = InsertionTS(
-                    model, x, y, metric="loss", baseline_mode=baseline_mode,
-                    steps=step, max_percentage_perturbed=max_percentage_perturbed,
-                )(explanations)
-                score_deletion = DeletionTS(
-                    model, x, y, metric="loss", baseline_mode=baseline_mode,
-                    steps=step, max_percentage_perturbed=max_percentage_perturbed,
-                )(explanations)
-
-                for score in [score_insertion, score_deletion]:
-                    assert 0.0 < score < 1
+    for step in [-1, 10]:
+        for max_percentage_perturbed in [0.2]:
```

Comment:
Why did you remove a max_percentage_pertubed value ? Especially the for loop is useless in this case ^^

Reply:
I had too much values, thus the test was too long and I though that if everyone does that, the testing may last for days in the end.
Thus I will remove the loop